src/main/java/by/exadel/internship/dto/locationDTO/CityDTO.java

```diff
@@ -7,11 +7,9 @@
 
 import java.util.UUID;
 @Data
-@Builder
 @NoArgsConstructor
 @AllArgsConstructor
 public class CityDTO {
 
-    private UUID id;
```

Comment:
Why did you remove id?  I think it'll be useful to have it

Reply:
I got you. Done